### DISCUSSION TITLE
Deployment Approvals - Required reviewers missing

### DISCUSSION THREAD

**Discussion by nod-dev** [community/community#30156] [Other Feature Feedback, Questions, & Ideas]
Hi,

I'm currently trying to set up a manual approval stage for a deployment, but for some reason there is no option to add required reviewers.
Does anyone have any idea of what may be causing the issue? The organisation has Github Team for all of its members, yet the environment only allows branch protection rules and secrets to be configured.

Any assistance would be massively appreciated.

Thanks.
<img width="1203" alt="image" src="https://user-images.githubusercontent.com/14308948/185714624-7ffd7b79-d12a-48bb-ae25-1866506437d2.png">

**Comment by tayeke** [comment]
I have the same issue all the docs say this feature exists and I literally cannot see it.

**Reply by nod-dev** [reply]
I eventually got a response from support on this. My issue was that I found the docs less than clear and I didn't read them correctly.

> Environments, environment secrets, and environment protection rules are available in public repositories for all products. For access to environments, environment secrets, and deployment branches in private or internal repositories, you must use GitHub Pro, GitHub Team, or GitHub Enterprise. For access to other environment protection rules in private or internal repositories, you must use GitHub Enterprise.

The key bit I missed was:
> For access to other environment protection rules in private or internal repositories, you must use GitHub Enterprise.

This was the reason only secrets and branches are available to me.

**Comment by alencarsouza** [comment]
[Required reviewers](https://docs.github.com/en/actions/managing-workflow-runs/reviewing-deployments) is definitely a feature which should at least be available in GitHub Team.

**Reply by Ghostbird** [reply]
Yeah, it makes no sense that you need Github Enterprise for it, given that a [pipeline step with a workaround functionality](https://github.com/marketplace/actions/manual-workflow-approval) is freely offered. The only disadvantage to the pipeline step is that it has a 72 hours timeout. In our case that's fine, if we're not going to approve within 72 hours, it's because we weren't planning to deploy to that environment anyway.

I'd get it if this was a feature request, and it was low-priority because the workaround is pretty good, so implementing it would add relatively little value. However the feature already exists. It's just arbitrarily locked behind an Enterprise subscription.

**Comment by jrmycanady** [comment]
Ouch, I upgraded to team for this single feature. I guess no reason to keep the upgrade...

**Comment by okvic77** [comment]
We need this feature for Team 🙌🏽

**Comment by ktalebian** [comment]
An alternative solution for now is to use https://github.com/trstringer/manual-approval as the final step in the previous environment

**Reply by partlythomas** [reply]
manual-approval by trstringer works, but it's a semi-dirty hack for something that at least the Teams plan should have!

**Comment by mmarqueee** [comment]
I also just upgrade to github team plan just for this feature :/

**Comment by Mington84** [comment]
Try double-checking the repository settings and make sure you have admin rights there. You could also check if the environment's settings are overriding other permissions, which sometimes happens. If it’s still not showing, it could be a limitation of the specific GitHub plan you're on.

**Reply by philouxx39** [reply]
XtreamIPTV cheat code

**Comment by Norbz** [comment]
I have a github pro licence, and this baffles me that this features requires an entreprise licence.

This is not about protecting an environment in a huge companies with dozens of developer, it's a very regular workflow : create release (like with release-please), release gets deployed to staging, release is approuved by a client -> deploy release to production

Current Workaround is either to duplicate the job put it in manual trigger, enter the release manually on an input, download the artifact, republish, or keep the same workflow, add a manual trigger, enter manually the tag...

Requiring a manual validation does not cost money to GH, this is just pure shitifications in the purpose of leveraging more money, but in freelancer's case, that's only makes me want to go back to gitlab.

**Comment by pensono** [comment]
This feature not being available to Team plans is very anti-small teams. My engineering team is just me so it makes no sense to use Enterprise yet I can't get the same protection features I need to move fast. Makes me excited for a GitHub alternative to pop up so I can switch!

**Comment by ayberkcanturk** [comment]
We are also thinking moving away from GitHub due to the same reason.

**Comment by Ciocotisan** [comment]
Well I wished I would not find this but... here it is... another scam.
Basically I get almost nothing usefull for my Team plan, not even a minimal and absolute normal feature...
